src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -189,8 +264,30 @@ const AcademiaSignUpForm = React.memo(({ setSuccess, success, ...props }) => {
                     choices={institutionTypes}
                     onChange={e => setInstitutionType(e)}
                 />
+                <StyledSelect
```

Comment:
There are a ton of options for countries, can we make the select scroll? Maybe set a max-height or something on it to allow `overflow` in css?

Reply:
Yeah, good advice. I will do that.